.travis.yml

```diff
@@ -19,6 +19,20 @@ jobs:
         - python -m compileall -q .
       env:
         - NAME="Lint Python script"
+    - stage: lint
+      os: linux
+      dist: bionic
+      language: python
+      python: 3.5
+      before_install:
+        - pip install flake8-putty==0.4.0
```

Comment:
Why are you installing into the python2 site-packages?

Reply:
Fixed to use python3